lib/cirro_io_v2/request_clients/jwt.rb

```diff
@@ -14,7 +14,6 @@ def initialize(base_url:, private_key:, client_id:)
         @connection = Faraday.new(url: base_url) do |conn|
           conn.request :json
           conn.response :json
-          conn.adapter :net_http
```

Comment:
@aliaksandr-volkau which routes were you calling? I didn't get any errors without this line 🤷‍♀️